include/nbl/builtin/shader/loader/gltf/common.glsl

```diff
@@ -0,0 +1,21 @@
+#define _NBL_V_IN_POSITION_ATTRIBUTE_ID 0
+#define _NBL_V_IN_NORMAL_ATTRIBUTE_ID 1
+#define _NBL_V_IN_UV_ATTRIBUTE_BEGINING_ID 2
+#define _NBL_V_IN_COLOR_ATTRIBUTE_BEGINING_ID 5
+#define _NBL_V_IN_JOINTS_ATTRIBUTE_BEGINING_ID 8
+#define _NBL_V_IN_WEIGHTS_ATTRIBUTE_BEGINING_ID 12
```

Comment:
I believe by "unwritten convention" normal goes into attribute 3 and color into 1

Also why gaps in the UV,Color and Joint attribute IDs?